core/src/graphman/core/copy.rs

```diff
@@ -0,0 +1,306 @@
+                deployment,
+                shard
+            )
+        })?;
+    store.activate(&deployment)?;
+
+    Ok(deployment)
+}
+
+pub fn list(
+    pools: HashMap<Shard, ConnectionPool>,
+) -> Result<
+    Vec<(
+        i32,
+        i32,
+        Option<DateTime<Utc>>,
+        DateTime<Utc>,
+        String,
+        Shard,
+    )>,
+    Error,
+> {
+    use catalog::active_copies as ac;
+    use catalog::deployment_schemas as ds;
+
+    let primary = pools.get(&*PRIMARY_SHARD).expect("there is a primary pool");
+    let conn = primary.get()?;
+
+    let copies = ac::table
+        .inner_join(ds::table.on(ds::id.eq(ac::dst)))
+        .select((
+            ac::src,
+            ac::dst,
+            ac::cancelled_at,
+            ac::queued_at,
+            ds::subgraph,
+            ds::shard,
+        ))
+        .load::<(i32, i32, Option<UtcDateTime>, UtcDateTime, String, Shard)>(&conn)?;
+
+    // LOG
+    // if copies.is_empty() {
+    //     println!("no active copies");
+    // } else {
+    //     fn status(name: &str, at: UtcDateTime) {
+    //         println!(
+    //             "{:20} | {}",
+    //             name,
+    //             at.to_rfc3339_opts(SecondsFormat::Secs, false)
+    //         );
+    //     }
+
+    //     for (src, dst, cancelled_at, queued_at, deployment_hash, shard) in copies {
+    //         println!("{:-<78}", "");
+
+    //         println!("{:20} | {}", "deployment", deployment_hash);
+    //         println!("{:20} | sgd{} -> sgd{} ({})", "action", src, dst, shard);
+    //         match CopyState::find(&pools, &shard, dst)? {
+    //             Some((state, tables, _)) => match cancelled_at {
+    //                 Some(cancel_requested) => match state.cancelled_at {
+    //                     Some(cancelled_at) => status("cancelled", cancelled_at),
+    //                     None => status("cancel requested", cancel_requested),
+    //                 },
+    //                 None => match state.finished_at {
+    //                     Some(finished_at) => status("finished", finished_at),
+    //                     None => {
+    //                         let target: i64 = tables.iter().map(|table| table.target_vid).sum();
+    //                         let next: i64 = tables.iter().map(|table| table.next_vid).sum();
+    //                         let done = next as f64 / target as f64 * 100.0;
+    //                         status("started", state.started_at);
+    //                         if log {
+    //                             println!(
+    //                                 "{:20} | {:.2}% done, {}/{}",
+    //                                 "progress", done, next, target
+    //                             )
+    //                         }
+    //                     }
+    //                 },
+    //             },
+    //             None => status("queued", queued_at),
+    //         };
+    //     }
+    // }
```

Comment:
It should be removed, since the logging logic has been integrated within the `list` command that extends the this core function in the graphman CLI.

Removed it!